plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -285,6 +286,27 @@ const eachMessage =
                             event.ip,
                             event.properties || {}
                         )
+                        // the replay record timestamp has to be valid and be within a reasonable diff from now
+                        if (replayRecord !== null) {
+                            const asDate = DateTime.fromSQL(replayRecord.first_timestamp)
```

Comment:
fromSQL ?

Reply:
https://github.com/moment/luxon/blob/master/docs/parsing.md#sql

That's the luxon method for parsing the "sql format" string we've created back to a date. So, if we're creating invalid dates instead of receiving them we'll find out

Reply:
Ah cool. Didn't know about that!